mslice/plotting/plot_window/interactive_cut.py

```diff
@@ -90,5 +91,5 @@ def flip_axis(self):
         self.plot_cut(*self.rect.extents)
 
     def window_closing(self):
+        self.slice_plot.interactive_cuts()
```

Comment:
Is this a toggle? It's not entirely clear from the name.

Reply:
Yes, it is. It toggles the interactive cut buttons and creates the icut object. `toggle_icut` is what creates the icut object. `toggle_icut_button` toggles the buttons. The two functions are within `interactive_cuts`. I could change it to `toggle_interactive_cuts`?

Reply:
Yes actually I think it would be worth tidying up to help with readability of the code.